composer.json

```diff
@@ -10,7 +10,8 @@
     "cakephp/plugin-installer": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.1.*"
+    "phpunit/phpunit": "4.1.*",
+    "loadsys/loadsys_codesniffer": "~3.0"
```

Comment:
That is, unless the intent is to run the codesniffer **on** the skeleton project itself, perhaps via Travis...